ts-tests/scripts/start-relay-and-para-chain.sh

```diff
@@ -8,6 +9,8 @@ POLKADOT_BIN=$GIT_ROOT/polkadot/target/release/polkadot
 
 # temp directory to store logs
 TMP_DIR="$1"
+TMP_DIR=${TMP_DIR:=/tmp}
```

Comment:
I think `:=` is unnecessary, or just 
```suggestion
TMP_DIR="${1:-/tmp}"
```